src/test/java/uk/gov/justice/digital/hmpps/keyworker/integration/ReturningFromLeaveIntTest.kt

```diff
@@ -0,0 +1,80 @@
+package uk.gov.justice.digital.hmpps.keyworker.integration
+
+import org.assertj.core.api.Assertions.assertThat
+import org.hibernate.envers.RevisionType
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.EnumSource
+import org.springframework.data.repository.findByIdOrNull
+import uk.gov.justice.digital.hmpps.keyworker.config.AllocationContext
+import uk.gov.justice.digital.hmpps.keyworker.config.AllocationContext.Companion.SYSTEM_USERNAME
+import uk.gov.justice.digital.hmpps.keyworker.config.AllocationPolicy
+import uk.gov.justice.digital.hmpps.keyworker.domain.StaffConfiguration
+import uk.gov.justice.digital.hmpps.keyworker.model.StaffStatus
+import java.time.LocalDate
+
+class ReturningFromLeaveIntTest : IntegrationTest() {
+  @ParameterizedTest
+  @EnumSource(AllocationPolicy::class)
+  fun `returning from leave does update due to return tomorrow`(policy: AllocationPolicy) {
```

Comment:
I think this should be "returning from leave does **not** update due to return tomorrow"